CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 Changelog
 =========
 
+# 5.2.0
+
+Added the following methods to the object returned from `fetchBuilder`:
+
+* `get` - makes a GET request using the behavior passed to fetchBuilder´
```

Comment:
markdown syntax should be wonky with the trailing `´`

Reply:
fixing